cmd/ko/commands.go

```diff
@@ -295,4 +320,113 @@ func addKubeCommands(topLevel *cobra.Command) {
 	addTagsArg(run, ta)
 
 	topLevel.AddCommand(run)
+
+	// TODO: Write docs
+	py := &cobra.Command{
+		Use:   "py -f FILENAME",
+		Short: "Print the input files with image references resolved to built/pushed image digests.",
+		Long:  `This sub-command finds import path references within the provided files, builds them into Go binaries, containerizes them, publishes them, and prints the resulting yaml.`,
```

Comment:
It's not clear to me after reading this how it's different from `ko resolve`

Reply:
Ah - I see the details here -  https://github.com/google/go-containerregistry/issues/379